packages/framework-provider-aws-infrastructure/src/infrastructure/params.ts

```diff
@@ -26,6 +28,8 @@ export function lambda(
       ...config.env, // Adds custom environment variables set by the user in the config file
       [environmentVarNames.restAPIURL]: baseURLForAPI(config, stack, apis.restAPI.restApiId),
       [environmentVarNames.websocketAPIURL]: baseURLForAPI(config, stack, apis.websocketAPI.ref),
+      [environmentVarNames.jwtIssuer]: userPool ? issuer(stack, userPool?.userPoolId) : '',
+      [environmentVarNames.jwksUri]: userPool ? jwksUri(stack, userPool?.userPoolId) : '',
```

Comment:
You don't need (as fas as I know) the question mark in these expressions as you already checked it is not null, so you can just do `userPool.userPoolId`

Reply:
Yup! my fault!